samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -764,6 +764,7 @@ class SamzaContainer(
         containerListener.afterStart()
       }
       metrics.containerStartupTime.set((System.nanoTime() - startTime)/1000000)
+      metrics.containerStarted.set(1)
```

Comment:
Not related to this PR, but curious what will be the behavior of the auto sizing controller if this metric is broken. Wondering if it can differentiate between metrics broken vs container not started.

Reply:
Without this (or another similar metric) the controller has no way of figuring out that a container has finished state-restore, upto which point any input-lag increase does not warrant scale-up